SPEC.md

```diff
@@ -82,30 +84,30 @@ Optional encryption:
 gpg --output reduce-worker.aci --digest-algo sha256 --cipher-algo AES256 --symmetric reduce-worker.aci
 ```
 
-All files in the image must maintain all of their original properties including: timestamps, Unix modes and extended attributes (xattrs).
+All files in the image must maintain all of their original properties, including timestamps, Unix modes, and extended attributes (xattrs).
 
 An image is addressed and verified against the hash of its uncompressed tar file, the _image ID_.
 The default digest format is sha512, but all hash IDs in this format are prefixed by the algorithm used (e.g. sha512-a83...).
 
 ```
-echo sha512-$(sha512sum reduce-worker.tar |awk '{print $1}')
+echo sha512-$(sha512sum reduce-worker.tar | awk '{print $1}')
 ```
 
 **Note**: the key distribution mechanism is not defined here.
 Implementations of the app container spec will need to provide a mechanism for users to configure the list of signing keys to trust or use the key discovery described in "App Container Image Discovery".
 
-Example application container image builder: **TODO** link to actool
+An example application container image builder is [actool](https://github.com/appc/spec/tree/master/actool).
 
 ### Image Manifest
 
-The [image manifest](#image-manifest-schema) is a JSON file that includes details about the contents of the app image, and optionally information about how to execute a process inside the app image's rootfs.
+The [image manifest](#image-manifest-schema) is a JSON file that includes details about the contents of the ACI, and optionally information about how to execute a process inside the ACI's rootfs.
 If included, execution details include mount points that should exist, the user, the command args, default cgroup settings and more.
 The manifest may also define binaries to execute in response to lifecycle events of the main process such as *pre-start* and *post-stop*.
 
 Image manifests MAY specify dependencies, which describe how to assemble the final rootfs from a collection of other images.
 As an example, you might have an app that needs special certificates layered into its filesystem.
-In this case, you can reference the name "example.com/trusted-certificate-authority" as a dependency in the app image manifest.
-The dependencies are applied in order and each app image dependency can overwrite files from the previous dependency.
+In this case, you can reference the name "example.com/trusted-certificate-authority" as a dependency in the image manifest.
+The dependencies are applied in order and each image dependency can overwrite files from the previous dependency.
 An optional path whitelist can be used to omit certain files from dependencies being included in the final assembled rootfs.
```

Comment:
I think this is just a quirk of wording. It is, strictly speaking, a whitelist: by only _explicitly_ permitting some files, any other files are _implicitly_ omitted. Does that make sense, or do you want to try reword to make it clearer?